index.md

```diff
@@ -98,11 +100,16 @@ Remember that even text---often thought of as a lowest common denominator---can
 
 1.  Every project has a short README file explaining its purpose
     *   Includes a contact address that actually works
-2.  And a LICENSE file
+2.  And a notes.txt file containing the to-do list and things people really need to know
```

Comment:
Could you please clarify "people"? Collaborators, anyone who wants to rerun the analysis, researchers who are just interested in the main results? This would also help to clarify the difference to the README file.